packages/core-mobile/app/services/send/SendService.ts

```diff
@@ -44,14 +45,6 @@ class SendService {
         }
         const service = this.getService(network, fromAddress)
 
-        sendState = await service.validateStateAndCalculateFees({
-          sendState,
-          isMainnet: !network.isTestnet,
-          fromAddress,
-          currency,
-          sentryTrx
-        })
```

Comment:
this is removed because we actually don't need to validate again when triggering a send (we have been validating it pre-send). this was also causing the slowness.